app/javascript/components/project/Project.vue

```diff
@@ -121,6 +130,8 @@
 <script>
 import _ from "lodash";
 import axios from "axios";
+import FileDownload from "js-file-download";
+import base64StringToBlob from "base64toblob";
```

Comment:
Can this dependency be removed here and in the package.json?

Reply:
good call

Reply:
done